src/target/target.c

```diff
@@ -688,3 +689,4 @@ int tc_system(target_s *t, target_addr_t cmd, size_t cmdlen)
 	}
 	return t->tc->system(t->tc, cmd, cmdlen);
 }
+#endif // PC_HOSTED
```

Comment:
While we probably should introduce a scheme like this onto the end of `#endif` statements, we don't currently use one, so please drop it, or at least use `/* */` comment style to match the header include guards

Reply:
Changed style as requested. I only try to mark `#endif` statements far below their corresponding `#if/#ifdef`s (more than 40 lines below), or in complicated nesting cases.

Reply:
Aye, we agree something needs to be done as there are some pretty large gaps in existing code between the preprocessor conditional and the closing `#endif` in some places.

If you want to go through all the code adding these in the style of `#endif /* PC_HOSTED */` in a separate PR, being this would be consistent with the style used for the include guards in the headers, this would be a welcome addition to the code base. Consistency is the key there.